Makefile.tools

```diff
@@ -0,0 +1,210 @@
+	@{ \
+		set -ex ;\
+		[[ -f $(KUSTOMIZE) ]] && exit 0 ;\
+		TMP_DIR=$$(mktemp -d) ;\
+		cd $$TMP_DIR ;\
+		go mod init tmp ;\
+		echo "Downloading kustomize" ;\
+		GOBIN=$(TOOLS_DIR) go get sigs.k8s.io/kustomize/kustomize/v3@$(KUSTOMIZE_VERSION) ;\
+		rm -rf $$TMP_DIR ;\
+	}
+
+.PHONY: operator-sdk
+$(OPERATOR_SDK) operator-sdk: $(TOOLS_DIR)
+	@{ \
+		set -ex ;\
+		[[ -f $(OPERATOR_SDK) ]] && exit 0 ;\
+		OS=$(shell go env GOOS) && ARCH=$(shell go env GOARCH) && \
+		curl -sSLo $(OPERATOR_SDK) https://github.com/operator-framework/operator-sdk/releases/download/$(OPERATOR_SDK_VERSION)/operator-sdk_$${OS}_$${ARCH} ;\
+		chmod +x $(OPERATOR_SDK) ;\
+	}
+
+.PHONY: opm
+$(OPM) opm: $(TOOLS_DIR)
+	@{ \
+		[[ -f $(OPM) ]] && exit 0 ;\
+		set -ex ;\
+		OS=$(shell go env GOOS) && ARCH=$(shell go env GOARCH) && \
+		curl -sSLo $(OPM) https://github.com/operator-framework/operator-registry/releases/download/$(OPM_VERSION)/$${OS}-$${ARCH}-opm ;\
+		chmod +x $(OPM) ;\
+	}
+
+.PHONY: promq
+$(PROMQ) promq: $(TOOLS_DIR)
+	@{ \
+		set -ex ;\
+		[[ -f $(PROMQ) ]] && exit 0 ;\
+		TMP_DIR=$$(mktemp -d) ;\
+		cd $$TMP_DIR ;\
+		echo "Downloading promq" ;\
+		git clone --depth=1 https://github.com/kubernetes-sigs/instrumentation-tools ;\
+		cd instrumentation-tools ;\
+		go build -o $(PROMQ) . ;\
+		rm -rf $$TMP_DIR ;\
+	}
+
+.PHONY: oc
+$(OC) oc: $(TOOLS_DIR)
+	@{ \
+		set -ex ;\
+		[[ -f $(OC) ]] && exit 0 ;\
+		OS=$(shell go env GOOS) && ARCH=$(shell go env GOARCH) && \
+		curl -sSLo $(OC) https://mirror.openshift.com/pub/openshift-v4/$${ARCH}/clients/oc/latest/$${OS}/oc.tar.gz ;\
+		tar -xf $(TOOLS_DIR)/oc -C $(TOOLS_DIR) ;\
+		rm -f $(TOOLS_DIR)/README.md ;\
+		$(OC) version ;\
+		version=$(OC_VERSION) ;\
+		$(OC) version | grep -q $${version##v} ;\
+	}
+
+.PHONY: crdoc
+$(CRDOC) crdoc: $(TOOLS_DIR)
+	@{ \
+		set -ex ;\
+		[[ -f $(CRDOC) ]] && exit 0 ;\
+		GOBIN=$(TOOLS_DIR) go install fybrik.io/crdoc@$(CRDOC_VERSION) ;\
+	}
+
+.PHONY: jsonnet
+$(JSONNET) jsonnet: $(TOOLS_DIR)
+		GOBIN=$(TOOLS_DIR) go install github.com/google/go-jsonnet/cmd/jsonnet@latest
+
+.PHONY: jsonnetfmt
+$(JSONNETFMT) jsonnetfmt: $(TOOLS_DIR)
+		GOBIN=$(TOOLS_DIR)  go install github.com/google/go-jsonnet/cmd/jsonnetfmt@latest
+
+.PHONY: jsonnet-lint
+$(JSONNET_LINT) jsonnet-lint: $(TOOLS_DIR)
+	GOBIN=$(TOOLS_DIR)  go install github.com/google/go-jsonnet/cmd/jsonnet-lint@latest
+
+.PHONY: jb
+$(JB) jb: $(TOOLS_DIR)
+	GOBIN=$(TOOLS_DIR) go install github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb@latest
+
+.PHONY: gojsontoyaml
+$(GOJSONTOYAML) gojsontoyaml: $(TOOLS_DIR)
+	GOBIN=$(TOOLS_DIR) go install github.com/brancz/gojsontoyaml@latest
+
+.PHONY: jsonnet-tools
+jsonnet-tools: jsonnet jsonnetfmt jsonnet-lint jb gojsontoyaml
+
+# Install all required tools
+.PHONY: tools
+tools: $(CONTROLLER_GEN) \
+ 		$(KUSTOMIZE) \
+		$(OC) \
+		$(OPERATOR_SDK) \
+		$(OPM) \
+	 	$(PROMQ) \
+	 	$(CRDOC) \
+	 	jsonnet-tools
+	@{ \
+		set -ex ;\
+		tools_file=.github/tools ;\
+		>$$tools_file ;\
```

Comment:
This misses the `### DO NOT EDIT ###` warning that was added recently in master .. is that on purpose?

Reply:
I think I removed it by accident, reverted.